python_ta/reporters/node_printers.py

```diff
@@ -389,23 +392,26 @@ def render_pep8_errors_e275(msg, line, col, source_line=None):
     )
 
 
-def render_pep8_errors_e301(msg, line, col, source_line=None):
+def render_pep8_errors_e301(msg, source_lines=None):
```

Comment:
This function doesn't need to use `source_lines`. Instead, it just needs to yield both the message (`NEW_BLANK_LINE_MESSAGE`) and the passed in `source_line`